ci/taos/checker-pr-gateway.sh

```diff
@@ -146,12 +147,12 @@ echo -e "[DEBUG] ./checker-pr-format-async.sh $1 $2 $3 $4 $5 $6 "    | tee -a $l
 echo -e "[DEBUG] Starting a format checker...            "           | tee -a $log_file_format
 # Run format checker
 pushd ./taos/
-./checker-pr-format-async.sh $1 $2 $3 $4 $5 $6                       | tee -a $log_file_format
+./checker-pr-format-async.sh $1 $2 $3 $4 $5 $6                       | tee -a $log_file_format &
 popd
 echo -e "[DEBUG] Running..."                                         | tee -a $log_file_format
 echo -e "[DEBUG] Completed."                                         | tee -a $log_file_format
```

Comment:
This will echo completed before completing the task?

Reply:
Probably ```echo -e "[DEBUG] Running..."   ``` is to be removed or moved to Line 150. (before checker-pr-format-async.sh)